client/run.go

```diff
@@ -194,6 +196,15 @@ func (cli *HyperClient) JsonFromCmdline(cmdArgs, cmdEnvs, cmdPortmaps []string,
 		ports = append(ports, *p)
 	}
 
+	for _, v := range cmdLabels {
+		eql := strings.Index(v, "=")
+		if eql > 0 {
+			labels[v[:eql]] = v[eql+1:]
+		} else {
+			labels[v] = ""
+		}
+	}
+
```

Comment:
I think we should not support the label format which does not contain "=".  And it will be better to delete the space if it has via `strings.TrimSpace(v)`.